.gitignore

```diff
@@ -40,3 +40,8 @@
 
 # Ignore application configuration
 /config/application.yml
+
+# Ignore all the DS_Stores
+/.DS_Store
```

Comment:
You can do simply `.DS_Store` without including a `/` or any directories. 

You will likely want to remove all the existing ones. `find . -name .DS_Store -delete` should do it

Reply:
Fixed it!